src/components/molecules/MoleculeArticleCard.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="mdc-list-item__start">
+    <div class="mdc-list-item__graphic">
+      <AtomIcon :src="src" />
```

Comment:
Change name of new component 
```suggestion
      <AtomProductAvatar :src="src" />
```